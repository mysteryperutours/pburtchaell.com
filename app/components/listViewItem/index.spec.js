import React from 'react';
import ListViewItem from './index';

describe(ListViewItem.name, () => {
  it('should render component', () => {
    expect(shallow(
      <ListViewItem />
    )).toMatchSnapshot();
  });
});
