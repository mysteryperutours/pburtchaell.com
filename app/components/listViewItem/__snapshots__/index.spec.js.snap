exports[`ListViewItem should render component 1`] = `
<Column
  className="list-view-item-container"
  elementType="div"
  size={6}
  style={Object {}}>
  <div
    className="list-view-item" />
</Column>
`;
